examples/asr/asr_chunked_inference/rnnt/speech_to_text_streaming_infer_rnnt.py

```diff
@@ -0,0 +1,470 @@
+
+    # if transcripts should not be overwritten, and already exists, skip re-transcription step and return
+    if not cfg.overwrite_transcripts and os.path.exists(cfg.output_filename):
+        logging.info(
+            f"Previous transcripts found at {cfg.output_filename}, and flag `overwrite_transcripts`"
+            f"is {cfg.overwrite_transcripts}. Returning without re-transcribing text."
+        )
+        return cfg
+
+    asr_model.freeze()
+    asr_model = asr_model.to(asr_model.device)
+    if cfg.compute_dtype != "float32":
+        asr_model.to(getattr(torch, cfg.compute_dtype))
+
+    # Change Decoding Config
+    with open_dict(cfg.decoding):
+        if cfg.decoding.strategy != "greedy_batch" or cfg.decoding.greedy.loop_labels is not True:
+            raise NotImplementedError(
+                "This script currently supports only `greedy_batch` strategy with Label-Looping algorithm"
+            )
+        cfg.decoding.preserve_alignments = False
+        cfg.decoding.fused_batch_size = -1  # temporarily stop fused batch during inference.
+        cfg.decoding.beam.return_best_hypothesis = True  # return and write the best hypothsis only
+
+    # Setup decoding strategy
+    if hasattr(asr_model, 'change_decoding_strategy'):
+        if not isinstance(asr_model, EncDecRNNTModel) and not isinstance(asr_model, EncDecHybridRNNTCTCModel):
+            raise ValueError("The script supports rnnt model and hybrid model with rnnt decodng!")
+        else:
+            # rnnt model
+            if isinstance(asr_model, EncDecRNNTModel):
+                asr_model.change_decoding_strategy(cfg.decoding)
+
+            # hybrid ctc rnnt model with decoder_type = rnnt
+            if hasattr(asr_model, 'cur_decoder'):
+                asr_model.change_decoding_strategy(cfg.decoding, decoder_type='rnnt')
+
+    feature_stride_sec = model_cfg.preprocessor['window_stride']
+    features_per_sec = 1.0 / feature_stride_sec
+    model_stride = cfg.model_stride
+    if manifest is not None:
+        records = read_manifest(manifest)
+    else:
+        assert filepaths is not None
+        records = [{"audio_filepath": audio_file} for audio_file in filepaths]
+
+    asr_model.preprocessor.featurizer.dither = 0.0
+    asr_model.preprocessor.featurizer.pad_to = 0
+    asr_model.eval()
+
+    decoding_computer: GreedyBatchedLabelLoopingComputerBase = asr_model.decoding.decoding.decoding_computer
+
+    audio_sample_rate = model_cfg.preprocessor['sample_rate']
+
+    features_frame2audio_samples = make_divisible_by(int(audio_sample_rate * feature_stride_sec), factor=model_stride)
+    encoder_frame2audio_samples = features_frame2audio_samples * model_stride
+
+    context_encoder_frames = ContextSize(
+        left=int(cfg.left_context_secs * features_per_sec / model_stride),
+        chunk=int(cfg.chunk_secs * features_per_sec / model_stride),
+        right=int(cfg.right_context_secs * features_per_sec / model_stride),
```

Comment:
can you use self.subsampling_factor where ever instead of calculating or passing it through config. We added these to models classes already,